features/waardetabel.feature

```diff
@@ -0,0 +1,248 @@
+      | Adellijke titel/predicaat (02.20) | <waarde>  |
+      En het systeem heeft landelijke tabel '38 Adellijke titel/predicaat' met de volgende gegevens
+      | code | omschrijving | soort     |
+      | BS   | barones      | titel     |
+      | JH   | jonkheer     | predicaat |
+      | R    | ridder       | titel     |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550001                       |
+      | fields              | naam.adellijkeTitelPredicaat    |
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
+      | naam                                 | waarde         |
+      | adellijkeTitelPredicaat.code         | <waarde>       |
+      | adellijkeTitelPredicaat.omschrijving | <omschrijving> |
+      | adellijkeTitelPredicaat.soort        | <soort>        |
+
+      Voorbeelden:
+      | waarde | omschrijving | soort     |
+      | BS     | barones      | titel     |
+      | JH     | jonkheer     | predicaat |
+      | R      | ridder       | titel     |
+      
+    Abstract Scenario: Omschrijving en soort bij code voor <relatie> Adellijke titel/predicaat (02.20)
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                              | waarde    |
+      | burgerservicenummer               | 555550001 |
+      En de persoon heeft een '<relatie>' met de volgende gegevens
+      | Adellijke titel/predicaat (02.20) | <waarde>  |
+      En het systeem heeft landelijke tabel '38 Adellijke titel/predicaat' met de volgende gegevens
+      | code | omschrijving | soort     |
+      | BS   | barones      | titel     |
+      | JH   | jonkheer     | predicaat |
+      | R    | ridder       | titel     |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550001                       |
+      | fields              | <relatie>.naam.adellijkeTitelPredicaat    |
+      Dan heeft de response een persoon een '<relatie>' met de volgende gegevens
+      | naam                                 | waarde         |
+      | adellijkeTitelPredicaat.code         | <waarde>       |
+      | adellijkeTitelPredicaat.omschrijving | <omschrijving> |
+      | adellijkeTitelPredicaat.soort        | <soort>        |
+
+      Voorbeelden:
+      | relatie | waarde | omschrijving | soort     |
+      | kind    | BS     | barones      | titel     |
+      | ouder   | JH     | jonkheer     | predicaat |
+      | partner | R      | ridder       | titel     |
+
+  Rule: wanneer voor de code geen bijbehorende waarde voorkomt in de tabel, wordt alleen de code geleverd
+
+    Abstract Scenario: geen omschrijving bij code voor <element>
```

Comment:
Het is niet zo dat er geen omschrijving bij deze code is. De codes van de voorbeelden komen helemaal niet voor in de tabellen. Dus volgens mij is het dan zo dat er een ongeldige code in <element> is opgenomen. 
Dat geeft overigens nog steeds wel de beschreven responses volgens mij.

Reply:
titel aangepast: code voor <element> komt niet voor in de tabel